### DISCUSSION TITLE
Curation Week 21 March

### DISCUSSION THREAD

**Discussion by elissonnog** [wikipathways/GPML-Repo#51] [Curation reports]
Hi Team,
Here is my curation report:

- Needs Work
WP5158, WP5210, WP5213, WP5214, WP5216, 
- Delete request
WP5091, 
- Fix Interactions
WP5215, 
- Approved Version
WP5200, WP5183, WP5189, WP5193, WP5171, WP5180, WP5182
WP5186, WP5191, WP5192, WP5194, WP5196, WP5202, WP5205,
WP5209, WP5211, WP138, WP4962, WP5173, WP5179
- Featured version
WP5169, WP5190, WP5178, WP5175, WP5188, WP5173, 

Removed a few deletion tags of Academic Test Pathways.

Sincerely,
Elisson

**Comment by khanspers** [comment]
Thank you @elissonnog !

**Comment by egonw** [comment]
Hi Elisson,

On Sat, 26 Mar 2022 at 18:56, E.N.Lopes ***@***.***> wrote:

>
>    - Approved Version
>    WP5200, WP5183, WP5189, WP5193, WP5171, WP5180, WP5182
>    WP5186, WP5191, WP5192, WP5194, WP5196, WP5202, WP5205,
>    WP5209, WP5211, WP138, WP4962, WP5173, WP5179
>
> I removed WP5209 again, because it did not have any identifiers; instead,
if just had the database name as identifier:

[image: image.png]

Greetings,

Egon

-- 
----
BiGCaT received a NWO Open Science grant to support our research into
interoperability of biological data and knowledge:
https://www.nature.com/articles/d41586-021-03418-1 and
https://www.nwo.nl/en/researchprogrammes/open-science/open-science-fund/open-science-fund-2021-awarded-grants

-----
E.L. Willighagen
Department of Bioinformatics - BiGCaT
Maastricht University (http://www.bigcat.unimaas.nl/)
Twitter/Mastodon: @egonwillighagen <https://twitter.com/egonwillighagen> /
@egonw ***@***.***>
Homepage: http://egonw.github.io/
Blog: http://chem-bla-ics.blogspot.com/
PubList: https://www.zotero.org/egonw
ORCID: 0000-0001-7542-0286 <http://orcid.org/0000-0001-7542-0286>
ImpactStory: https://impactstory.org/u/egonwillighagen
